<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CheckBoxes</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>16</number>
    </property>
	<item row="0" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Группа 1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Пункты не выбраны</string>
      </property>
     </widget>
    </item>
	<item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Группа 2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Пункты не выбраны</string>
      </property>
     </widget>
    </item>
	<item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Группа 3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Пункты не выбраны</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>92</y>
     <width>147</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбор пунктов</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
